pkg/operator/config/config.go

```diff
@@ -17,10 +17,11 @@ limitations under the License.
 package config
 
 const (
-	DefaultExternalDNSImage  = "docker.io/bitnami/external-dns:latest"
-	DefaultMetricsAddr       = "127.0.0.1:8080"
-	DefaultOperatorNamespace = "externaldns-operator"
-	DefaultOperandNamespace  = "externaldns"
+	DefaultExternalDNSImage     = "docker.io/bitnami/external-dns:latest"
```

Comment:
We should use a quay.io pullspec, but maybe that should be in a follow-up.

Reply:
I'd like to do it too but I put on hold the setup of the mirroring for ART built external-dns images.